src/test/java/com/checkmarx/flow/cucumber/integration/sca_scanner/ScaCommonSteps.java

```diff
@@ -46,4 +47,16 @@ protected ScanRequest getBasicScanRequest(String projectName, String repoWithAut
                 .map(String::trim)
                 .collect(Collectors.toList());
     }
+
+    protected long getTotalHighFindings(SCAResults scaResults) {
```

Comment:
These methods are only used in ScaZipScanSteps, so it's better to move them there.
getTotalHighFindings and getTotalMediumFindings can be combined into a single method that also includes an assertion.

```
private void validateFindingCount(SCAResults scaResults, String severity) {
   long count = ...
    Assert.assertTrue("...", count > 0);
}
```

Reply:
fixed.